src/lib/Scenes/Artwork/Components/ArtworkActions.tsx

```diff
@@ -77,12 +78,14 @@ export class ArtworkActions extends React.Component<ArtworkActionsProps> {
             </UtilButton>
           </TouchableWithoutFeedback>
           {Constants.AREnabled && (
-            <UtilButton pr={3}>
-              <Box mr={0.5}>
-                <EyeOpenedIcon />
-              </Box>
-              <Sans size="3">View in Room</Sans>
-            </UtilButton>
+            <TouchableWithoutFeedback onPress={() => ApiModule.presentAugmentedRealityVIR("url", 1, 2, "slug", "id")}>
```

Comment:
Should these be passing in an actual `url`, `slug`, and `id` vs. strings?

Reply:
Yeah good call. Maybe we could even modify the ObjC alert to show the data that's passed into the `presentAugmentedRealityVIR` function.

Reply:
That would be doable with the [`NSString` method `stringWithFormat:`](https://developer.apple.com/documentation/foundation/nsstring/1497275-stringwithformat). Something like:

```objc
...
message: [NSString stringWithFormat:@"slug %@, width %f, height %f", slug, width, height]
...
```

Basically a C `printf`-style format string, with `%@` calling ObjC's equivalent of `toString()`.